lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -305,9 +305,11 @@ def self.resolve_automation_object(uri, user_obj, attr = nil, options = {}, read
 
   def self.ae_user_object(options = {})
     raise "user_id not specified in Automation request" if options[:user_id].blank?
-    # raise "group_id not specified in Automation request" if options[:miq_group_id].blank?
+    raise "miq_group_id not specified in Automation request" if options[:miq_group_id].blank?
```

Comment:
@lfu Do you know why this was commented out before?  Are there any code paths you are aware of that might run into this new requirement?